.github/workflows/cibuildwheel-before-build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+if [ -z $1 ]; then
+    echo "A package directory must be provided as the first argument."
+    exit 1
+fi
+if [ -z $2 ]; then
+    echo "A platform {linux,macos} must be provided as the second argument."
+    exit 1
+fi
+
+# Install a modern version of CMake for compatibility with TBB 2021
+# (manylinux image includes CMake 2.8)
+pip install cmake
+
+PACKAGE_DIR=$1
+PLATFORM=$2
+TBB_VERSION="2021.2.0"
+#if [[ "${TBB_PLATFORM}" == "win" ]]; then
+#    TBB_ZIP="oneapi-tbb-${TBB_VERSION}-${TBB_PLATFORM}.zip"
+#else
+#    TBB_ZIP="oneapi-tbb-${TBB_VERSION}-${TBB_PLATFORM}.tgz"
+#fi
+TBB_ZIP="v${TBB_VERSION}.zip"
+#curl -L -O "https://github.com/oneapi-src/oneTBB/releases/download/v${TBB_VERSION}/${TBB_ZIP}"
+curl -L -O "https://github.com/oneapi-src/oneTBB/archive/refs/tags/${TBB_ZIP}"
+unzip -q "${TBB_ZIP}"
+
+# Move to a hard-coded path (defined by CIBW_ENVIRONMENT)
+mv "oneTBB-${TBB_VERSION}" "${PACKAGE_DIR}/tbb"
+cd "${PACKAGE_DIR}/tbb"
+mkdir -p build
+cd build
+cmake ../ -DTBB_TEST=OFF
+cmake --build . -j
+cmake -DCOMPONENT=runtime -P cmake_install.cmake
+cmake -DCOMPONENT=devel -P cmake_install.cmake
```

Comment:
```suggestion
cmake --install .
```
Is this a documented method to install TBB from source?

I use `cmake --install` in `glotzerlab-software` and it works, provided I set an installation path with `-DCMAKE_INSTALL_PREFIX=` in the first `cmake` invocation. This installs both headers and libs in standard `<PREFIX>/include` and `<PREFIX>/lib` directories.

You may need to adjust your `CMAKE_PREFIX_PATH` and `LD_LIBRARY_PATH` if you make this change.

Reply:
I think I found this option and chose to use it because it was cross-platform and didn't require me to know which make tool to use. I don't really know what to say about `CMAKE_PREFIX_PATH` or `LD_LIBRARY_PATH` but I would welcome a suggestion if you think there is a simpler way (perhaps one that avoids the workaround in the CMake file to determine `INSTALL_RPATH_USE_LINK_PATH` based on the environment variable `CIBUILDWHEEL`).

Reply:
`cmake --install .` is platform independent (like `cmake --build` it forwards the install command to whatever tool it generated build files for). However, it will not remove the need for `INSTALL_RPATH_USE_LINK_PATH`. It will just put the files under a standard `include/ lib/` hierarchy under $PREFIX.

Reply:
Oh, sorry. I was confused. I misread the original comment and thought your code suggestion _was_ the current code, so I didn't understand what you were trying to indicate. I'll try your suggestion now.